app-frontend/packs/components/baseComponents/Input.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+
+    <input
+        class="tt-input"
+        :type="type"
+        :value="modelValue"
+        @input="updateInput"
+    >
+
+</template>
+
+<script>
+
+export default {
+    name: "Input",
+    props: {
+        modelValue: [String, Number],
+        type: String,
+        required: false,
+        default: false
+    },
```

Comment:
что-то должно получится типа такого:

Если ты тайп всегда передаешь, то указывай значение `required: true` и `default` тогда не нужен

Если это пропс, который можно передавать, а можно и нет, то указывай `required: false` и тогда передавай `default`

```suggestion
    props: {
        modelValue: {
            type: [String, Number],
            required: true
        },
        
        type: {
            type: String,
            required: true
        }
    },
```

Reply:
исправил